src/include/pka_cpu.h

```diff
@@ -79,17 +85,212 @@ static inline uint64_t pka_early_cpu_speed(void)
   return CPU_HZ_MAX;
 }
 #else
-
+#define PKA_SYS_SMBIOS_ENTRY_POINT "/sys/firmware/dmi/tables/smbios_entry_point"

```

Comment:
Style issue: Could you please align the macro values, for example:

  #define SHORT_MACRO                 x  
  #define VERY_LONG_MACRO         y

Reply:
For some reason the values don't appear aligned. Bu you got what I mean right?

Reply:
Please also group defines with same prefix together and keep each group separate (by an empty line)

Reply:
Yup, got it. Thanks